### PR TITLE
Updated Readme file

### PR DESCRIPTION
**Pull request by monika-soam**
**Interactive List**

- User can mark task completion by selecting the corresponding checkbox or undo it by unchecking. 
- The updated tasks list will be stored in local storage.
- Added event listener to the checkbox (change).
- Updated items object's value for completed key upon user actions.
- No linters error, followed GitHub flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## To-do-list ![To-do-list](https://img.shields.io/badge/Microverse-blueviolet)
 
 
-To do list is a basic website, On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task also implemented the CRUD (create, update, delete) methods.
+To do list is a task mangement webapp, On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task also implemented the CRUD (create, update, delete) methods.The user will also be able to mark task completion clicking on checkbox and undo it.
```
Comment (BenMukebo) [inline README.md:5]: - [x] Please use the `change` event listener on the checkbox as per the requirement, actually, you have used the click event listener, kindly change that
chk

- [x] Consider using the `filter` array method to filter all completed items on the `removeAllcompleted function which is in your case this below ⬇️ 
filter

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi ** @monika-soam, **

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 

 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (monika-soam) [reply README.md:5]: Updated the changes required

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @monika-soam, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**